src/T2DMap.cpp

```diff
@@ -2093,7 +2097,7 @@ void T2DMap::paintRoomExits(QPainter& painter, QPen& pen, QList<int>& exitList,
             }
         }
 
-        for (const int& k : exitList) {
+        for (const int& k : qAsConst(exitList)) {
```

Comment:
This was detaching and creating a copy of existList each time we were iterating, which is pricey on a million room map.

Reply:
But was it though, given Qt's liking of COW-ing - we are not modifying `exitList` are we so we shouldn't be detaching...? OTOH it doesn't seem harmful to make this change.

Reply:
We were unfortunately detaching and this was causing a 15% slowdown, even on rooms where the list of exits was zero.

Reply:
Surprising, but 🤷‍♂️ . Okay.